tools/loader/ImageClassifier.cpp

```diff
@@ -135,11 +157,17 @@ int main(int argc, char **argv) {
 
   // Load and process the image data into the data Tensor.
   Tensor data;
-  loadImagesAndPreprocess(inputImageFilenames, &data, imageMode);
+
+  loadImagesAndPreprocess(inputImageFilenames, &data, imageMode, loadimageFormat);
 
   // All of our ONNX and Caffe2 models may use one of two names for input.
   constexpr char const *c2Inputs[2] = {"gpu_0/data", "data"};
-  constexpr char const *onnxInputs[2] = {"gpu_0/data_0", "data_0"};
+  
+  char const *onnxInputs[2] = {"gpu_0/data_0", "data_0"};
+  if(loadimageFormat == "NHWC") {
```

Comment:
I imagine there will be models with different input names, maybe worthwhile passing this (inputs) as a CLI parameter.
If the parameter is missing then use that default ```{"gpu_0/data_0", "data_0"};```

Reply:
Can this be a separate change list? I do not want to combine this as part of this pull request. Is that ok? I believe there can be more underlying changes for this.

Reply:
Sure, feel free to do this on separate PR. Smaller PRs are easier to review :)